src/geometry/algebra/lie_group.lean

```diff
@@ -0,0 +1,258 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Nicolò Cavalleri.
+-/
+
+import geometry.manifold.times_cont_mdiff
+
+noncomputable theory
```

Comment:
Move this after the file-level docstring

Reply:
If I do this it says `invalid 'import' command, it must be used in the beginning of the file`

Reply:
Just the line `noncomputable theory` should be moved. The beginning of a file should look like:
header
imports
file-level docstring
everything else